File: ViewModels/EmailMessageToPractitionerViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KPProject.ViewModels
{
    public class EmailMessageToPractitionerViewModel
    {
        public string PractitionerId { get; set; }
        public string MessageContent { get; set; }
        public string MessageSubject { get; set; }
    }
}

File: ClientApp/src/view-models/email-message-to-practitioner-view-model.ts
export class EmailMessageToPractitionerViewModel {
  public practitionerId: string;
  public messageContent: string;
  public messageSubject: string;

  constructor(practitionerId, messageSubject, messageContent) {
    this.practitionerId = practitionerId;
    this.messageSubject = messageSubject;
    this.messageContent = messageContent;
  }
}
